### ISSUE TITLE
Missing constants ASC and DSC

### ISSUE THREAD

**Issue by haslinger** [activerecord-hackery/ransack#705]
Updating to Rails 5 I came across the issue, that the Constants 
```
module Ransack
  module Constants
    # ...
    ASC = 'asc'.freeze
    DESC = 'desc'.freeze
  end
end
```
are missing in Ransack::Constants.

I'm not sure if that's intentional, so I now monkey patched them into the demo app.
See my pull request https://github.com/activerecord-hackery/ransack_demo/pull/11 for details.

**Comment by jonatack** [comment]
Thanks. This was fixed by aece23c in Ransack master, will make a new release with it.

**Comment by haslinger** [comment]
Ups, sorry. Running on master, but didn't update 😊.
